.github/workflows/test.yml

```diff
@@ -38,7 +38,8 @@ jobs:
         --eval "(setq debug-on-error t)" --load init.el
       name: Test if package.el works inside Emacs
     - run: |
-        nix-build --argstr emacsAttr emacs-${{ matrix.attr }} tests \
+        nix-build --experimental-features 'nix-command flakes' \
+                  --argstr emacsAttr emacs-${{ matrix.attr }} tests \
```

Comment:
Now that you are going to have a flake, you can define the tests under `checks` output of the flake, so you can run each test using the following command:

```
nix build .#checks.x86_64-linux.test-emacs-27-1
```

You can build and test all packages with a single command on your machine:

```
nix flake check
```

Reply:
Ooh, that would be nice.